content/docs/next/dev-internal/release.md

```diff
@@ -36,7 +36,7 @@ All releases version numbers follow the format of [semantic versioning 2.0.0](ht
 - Ensure the relevant milestone on GitHub is complete. All referenced issues should be closed, or moved elsewhere.
 - Remove this release from [roadmap](https://github.com/etcd-io/etcd/blob/master/ROADMAP.md), if necessary.
 - Ensure the latest upgrade documentation is available.
-- Bump [hardcoded MinClusterVerion in the repository](https://github.com/etcd-io/etcd/blob/master/version/version.go#L29), if necessary.
+- Bump [hardcoded MinClusterVerion in the repository](https://github.com/etcd-io/etcd/blob/v3.4.15/version/version.go#L29), if necessary.
```

Comment:
Here and elsewhere, I'd vote against hardcoding version numbers. Rather, we should use a front-matter parameter to specify the latest tagged version (`v3.4.15` in this case) and then reference that parameter in the markdown source, something like this:

    .../blob/{{< param version_tag >}}/...

(the param name is just an example / possible suggestion)

Reply:
This is an interesting idea. I think I'd rather put the latest tagged versions in the config though, putting them in the front matter would mean someone may still need to change 10s of files for updates.

Reply:
> ... putting them in the front matter would mean someone may still need to change 10s of files for updates.

Not if you use the cascade feature, as we are currently for the `version` field.

Reply:
That's an idea. I'll try that.

Reply:
I've done this for all versions except `next`/`v3.5.0-alpha.0` as it's changed the structure of the repo enough that some files have been moved/removed. We'll need to figure out where they're going to point now manually (`/docs/next/dev-internal`).

Reply:
See #252